stingray/tests/test_crosscorrelation.py

```diff
@@ -75,6 +77,23 @@ def test_corr_is_correct(self):
         assert cr.mode == 'same'
         assert cr.auto is False
 
+    def test_crossparam_input(self):
+        # need to create new results to check against
+        spec = Crossspectrum(self.lc1, self.lc2, fullspec=True)
+        ifft = abs(scipy.fft.ifft(spec.power).real)
+        time = scipy.fft.fftfreq(len(ifft), spec.df)
+        time, resultifft = (list(t) for t in zip(*sorted(zip(time, ifft))))
+        cr2 = CrossCorrelation(cross=spec)
+
```

Comment:
Need to insert `lags_result = np.array([-2, -1, 0, 1, 2])` on a line up here, since lags_result isn't defined in this method and that's why the tests fail